src/less/tables.less

```diff
@@ -184,9 +184,9 @@
   -ms-flex-direction: column;
   flex-direction: column;
 
-  .rt-tr.-odd {
+  .rt-tr.-even {
     & + .expand-row {
-      background-color: rgba(0, 0, 0, 0.03);
+      background-color: #f7f7f7;
```

Comment:
I know this is a new feature but is there a reason we can't use existing color variables? I see `#f7f7f7` and in the existing variables file we have `@global-color-grey-lightest` which is `#f9f9f9`. Both are almost identical. 

I understand if it was requested exactly that color, but if we have flexibility I'd recommend keeping things consistent whenever possible.

Reply:
@juanfabrega Yes, this can be done. I have updated the color.